packages/components-js/tests/e2e/puppeteer/specs/__snapshots__/default-dom.e2e.ts.snap

```diff
@@ -297,6 +297,31 @@ exports[`should have no basic DOM regression for p-link-tile-model-signature 1`]
 </div>"
 `;
 
+exports[`should have no basic DOM regression for p-link-tile-product 1`] = `
+"<div class="root">
+  <div id="header" class="header">
+    <slot name="header">
+    </slot>
+    <p-button-pure class="like-button hydrated">
+      Heart icon</p-button-pure>
+  </div>
+  <div class="image-container">
```

Comment:
Shorter css class names would also be fine, e.g. `image`.

Reply:
image-container and link-overlay are used to be consistent with link-tile